index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Document</title>
+  <link rel="stylesheet" href="css/style.css" />
+  <script src="https://kit.fontawesome.com/5784b9ea6f.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <div class="wrapper">
+    <header>
+      <nav class="flex-nav">
+        <h1 class="logo"><a href="#">My Logo</a></h1>
```

Comment:
I suggest you take away the `h1` tag wrapping your logo as this makes your page more semantic😉.

```suggestion
        <a href="#" class="logo">>My Logo</a>
```